aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/scss/_noUISlider.scss

```diff
@@ -153,7 +153,6 @@
         height: 14px;
         width: 1px;
         background: #E8E7E6;
-        left: 14px;
```

Comment:
why is this removed?

Reply:
Because it is useless. when there is a two same property in css class, only last is set to value so. second already overrided and sonarqube shows error message.